### PR TITLE
Feature/tr 5801/UI event log alignment

### PR DESCRIPTION
**Pull request by marpesia**
**Related to**: [TR-5801](https://oat-sa.atlassian.net/browse/TR-5801)

**Description:**
Include `autostart` metadata even if the delay is setup

[TR-5801]: https://oat-sa.atlassian.net/browse/TR-5801?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by btamas — APPROVED** [review]
Code looks straight forward. taoUpdate runs without problem. Event is fired in preview.
Screenshot 2023-10-11 at 08 43 35


**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js [hunk 1]
```diff
@@ -628,7 +628,7 @@ define([
                         self.startRecording();
                         dispatchInteractiontraceEvent({
                             domEventType: 'record',
-                            auto: true,
+                            autostart: true,
```
[the remark this answers is not in the document]
Comment (marpesia) [reply views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js:631]: all `auto` changed to `autostart`

**Review by btamas — APPROVED** [review]
Now, everything looks good :)

**Comment by btamas** [comment]
@marpesia Please add the correct `componentType` to the events. It should be `Custom Audio PCI`

**Pull request merged**